cves/kernel/CVE-2022-27223.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: In a section of the Linux kernel code related to USB drivers (specifically getting the status and setting/clearing features), lack of validation on the endpoint index of an array could potentially lead it to be manipulated beyond the intended bounds by the host system. This manipulation might lead to accessing data outside the intended array, creating a security concern.
```

Comment:
While the instructions state to refrain from using a lot of jargon, I think this is a little too vague.  What concerns?  What sort of manipulations would result in these concerns?